dnsviz/analysis/errors.py

```diff
@@ -1260,7 +1260,7 @@ class NotAuthoritative(ResponseError):
     '''
 
     _abstract = False
-    code = 'NOT_AUTHORITATVE'
+    code = 'NOT_AUTHORITATIVE'
```

Comment:
This might be an api break

Reply:
Good point, but I think this one is okay.